### DISCUSSION TITLE
如何与本网站的登录系统同步 || How to synchronize with the login system of this website

### DISCUSSION THREAD

**Discussion by Npc-h** [walinejs/waline#2038] [Q&A] [answered]
想把waline整合到自身的网站，但自身的网站已经有登录注册了，如何与自身的网站实现用户同步？

---
I want to integrate waline into my own website, but my own website has already been logged in and registered. How to realize user synchronization with my own website?

**Comment by lizheming** [comment, marked as answer]
https://waline.js.org/cookbook/customize/userdb.html
